pkg/cmd/pipeline/start.go

```diff
@@ -358,38 +358,6 @@ func (opt *startOptions) getInput(pipeline *v1beta1.Pipeline) error {
 		}
 	}
 
-	if opt.UseParamDefaults && !opt.Last && opt.UsePipelineRun == "" {
-		if err = opt.includeDefaultParams(pipeline); err != nil {
-			return err
-		}
-	}
-
-	return nil
```

Comment:
I think we need this

Reply:
We have discussed this thing in PR #1036 to proceed with the same way we have done for task start [here](https://github.com/tektoncd/cli/blob/master/pkg/cmd/task/start.go#L530) and correct that in pipeline start command.

Reply:
I am saying for just one line `return nil` not the whole block, I see it there in the above-mentioned PR also

Reply:
Okay, will add this. That was by mistake :sweat_smile: , thanks for the review :smile: